trade-finance-manager-ui/component-tests/utilisation-reports/add-to-an-existing-payment.component-test.ts

```diff
@@ -94,23 +94,23 @@ describe(page, () => {
     wrapper.expectText('[data-cy="continue-button"]').toRead('Continue');
   });
 
-  it('should render the cancel link', () => {
+  it('should render the cancel link with the selected fee record ids', () => {
     // Arrange
     const addToAnExistingPaymentViewModel = anAddToAnExistingPaymentViewModel();
     const wrapper = render(addToAnExistingPaymentViewModel);
 
     // Assert
     wrapper.expectElement('[data-cy="cancel-link"]').toExist();
-    wrapper.expectText('[data-cy="cancel-link"]').toRead('Cancel');
+    wrapper.expectLink('[data-cy="cancel-link"]').toLinkTo('/utilisation-reports/12?selectedFeeRecordIds=456', 'Cancel');
```

Comment:
This link looks magic as far as this test is concerned, please can we fields that are relevant to the assertions in the test itself? In fact, the template knows nothing about selected fee record ids, that is determined by the mapper mapping to the view model, so the title of the test should change to something like `should render cancel link button linking to backLinkHref` (same thing for test below)

Reply:
Would be nice to use a mock variable for the IDs as well e.g `mockReportId` `mockFeeRecordId`